packages/typespec-test/test/widget_dpg/generated/typespec-ts/src/models/index.ts

```diff
@@ -8,8 +8,8 @@ export {
   CreateWidget,
   UpdateWidget,
   AnalyzeResult,
-  NonReferencedModel,
   Versions,
```

Comment:
when can we fix the unrelated changes caused by the order change from tcgc or the emitter?